src/contrast/pages/UploadNewPaper/ExperimentsSection/Findings.jsx

```diff
@@ -97,9 +97,7 @@ export default function Findings({
       return false;
     }
     if (field?.family == families["Temporal"]) {
-      return [field.onset, field?.offset].every(
-        (condition) => Boolean(condition) === true
-      );
+      return Boolean(field.onset) === Boolean(field.offset);
```

Comment:
this is wrong and not the same as the previous one - you (almost) never have both fields the same value. you need to check both "exist" ,